### PR TITLE
Better messages for the rc3 migration stuff

### PR DESCRIPTION
**Pull request by tailhook**

### CODE REVIEW

File: src/cli/install.rs [hunk 1]
```diff
@@ -205,6 +205,41 @@ fn ensure_line(path: &PathBuf, line: &str) -> anyhow::Result<()> {
...
 }
 
+fn print_post_upgrade_rc3_message()
+{
+    if cfg!(windows) {
+        print_markdown!("\
+            # The EdgeDB command-line tool is now upgraded!\n\
+            \n\
+            We have changed how server is installed and instances are
+            set up. Unfortunately this requires some manual steps to convert\n\
```
Comment (elprans) [inline src/cli/install.rs:215]:
```suggestion
            set up. A few manual steps are needed to convert\n\
```

File: src/cli/install.rs [hunk 2]
```diff
@@ -205,6 +205,41 @@ fn ensure_line(path: &PathBuf, line: &str) -> anyhow::Result<()> {
     Ok(())
 }
 
+fn print_post_upgrade_rc3_message()
+{
+    if cfg!(windows) {
```
Comment (elprans) [inline src/cli/install.rs:210]: Inverted condition?

File: src/portable/control.rs [hunk 3]
```diff
@@ -16,7 +16,8 @@ pub fn fallback(name: &str, success_message: &str,
           "Looking for deprecated instances...");
     crate::server::control::instance_command(cmd)?;
     eprintln!("{}", success_message);
-    print::warn("Please upgrade instance to portable installation");
+    print::warn("Please convert the instance to use portable installation.");
```
Comment (elprans) [inline src/portable/control.rs:19]:
```suggestion
    print::warn("Please convert the instance to the new installation layout.");
```

File: src/portable/status.rs [hunk 4]
```diff
@@ -364,9 +383,9 @@ pub fn list(options: &List) -> anyhow::Result<()> {
         print_table(&local, &remote);
     }
 
-    echo!("Only portable packages shown here, \
-        use `--deprecated-install-methods` \
-        to show docker and package installations.".fade());
+    echo!("Older instances may be listed here as `remote`. \
```
Comment (elprans) [inline src/portable/status.rs:386]:
```suggestion
    echo!("Instances from legacy installation methods may be listed here as `remote`. \
```

File: src/cli/install.rs [hunk 5]
```diff
@@ -205,6 +205,41 @@ fn ensure_line(path: &PathBuf, line: &str) -> anyhow::Result<()> {
...
+            # The EdgeDB command-line tool is now upgraded!\n\
+            \n\
+            We have changed how server is installed and instances are
+            set up. Unfortunately this requires some manual steps to convert\n\
+            your instances if there are any. To list instances that need\n\
+            to be converted, run: \n\
+            ```\n\
+                edgedb instance list --deprecated-install-methods\n\
+            ```\n\
+            Migration process and details: https://edgedb.com/p/rc3-upgrade
```
Comment (elprans) [inline src/cli/install.rs:221]:
```suggestion
            For migration process details see https://edgedb.com/p/rc3-upgrade
```

**Review by elprans — APPROVED** [review]

File: src/cli/install.rs (hunk 2 above)
Comment (tailhook) [reply src/cli/install.rs:210]: Oh, right!

**Pull request merged**